docs/cse/integrations/automation-service.md

```diff
@@ -1,172 +1,263 @@
-* The nslookup Automation is configured to only run on IP Addresses so it will run three times.
-* No Automation will run on the Hostname.
+   * The CarbonBlack automation is configured for IP Addresses, Email Addresses, and Domain Names, so it will run four times (once for the Email Address and once for each IP Address selected on the previous screen).
+   * The nslookup automation is configured to only run on IP Addresses so it will run three times.
+   * No automation will run on the Hostname.
 
-## Viewing the Status of Automations
+### View an automation's status
 
-After running an Automation, you can go to the Automations tab to view its status.  
+After [running an automation](#step-4-view-results-of-an-automation), you can go to the **Automations** tab for the Insight or Entity to view the automation's  status.  
 
-<img src={useBaseUrl('img/cse/automations-execution-status.png')} alt="Automations execution status" width="600"/>
+<img src={useBaseUrl('img/cse/automations-execution-status.png')} alt="Automations execution status" width="800"/>
 
-The list of Automations is organized by Insight and Entity, and each section can be collapsed and expanded. On each card you will find:
-* The time and date when the Automation was run.
-* The name and description of the associated Playbook.
-* The Playbook’s current status.
-* A link to **View Playbook** in the CSE Automation Service UI.
+On each card you will find:
+* The time and date when the automation was run.
+* The name and description of the associated playbook.
+* The playbook’s current status.
+* A link to **View Playbook** in the Automation Service UI.
 
 :::note
-During the Beta, you may have to manually refresh this screen to see the most current Status.
+You may have to manually refresh this screen to see the most current status.
 :::
 
-If you click **View Playbook**, the CSE Automation Service UI will open to the Playbook Status page:
+If you click **View Playbook**, the Automation Service UI will open to the playbook status page:
 
 <img src={useBaseUrl('img/cse/automations-playbook-status.png')} alt="Playbook status" width="600"/>
 
 You can switch to the graphical view by clicking **Graph** in the upper-right corner:
 
 <img src={useBaseUrl('img/cse/automations-playbook-status-graph.png')} alt="Playbook status graph" width="600"/>
 
-For more information about the Playbook Status page and understanding how to interact with the Playbook graph, see [Cloud SOAR Automation](/docs/cloud-soar/automation/).
+### Enrichments and threat indicators
 
-## Enrichment and Threat Indicators
+You can view the results of enrichments in Cloud SIEM by navigating to the newly-redesigned **Enrichments** tab (which will appear on the Entity, Signal, and Insight details pages if there are any enrichments to display):
```

Comment:
In six months, it won't be **newly redesigned**. Removing it keeps the content evergreen.

```suggestion
You can view the results of enrichments in Cloud SIEM by navigating to the **Enrichments** tab (which will appear on the Entity, Signal, and Insight details pages if there are any enrichments to display):
```